server/src/main/java/io/seata/server/lock/DefaultLockManager.java

```diff
@@ -38,105 +29,16 @@
  */
 public class DefaultLockManager extends AbstractLockManager {
 
-    private static Locker locker = null;
-
-    /**
-     * The constant CONFIG.
-     */
-    protected static final Configuration CONFIG = ConfigurationFactory.getInstance();
-
-    @Override
-    public boolean acquireLock(BranchSession branchSession) throws TransactionException {
-        if (branchSession == null) {
-            throw new IllegalArgumentException("branchSession can't be null for memory/file locker.");
-        }
-        String lockKey = branchSession.getLockKey();
-        if (StringUtils.isNullOrEmpty(lockKey)) {
-            //no lock
-            return true;
-        }
-        //get locks of branch
-        List<RowLock> locks = collectRowLocks(branchSession);
-        if (CollectionUtils.isEmpty(locks)) {
-            //no lock
-            return true;
-        }
-        return getLocker(branchSession).acquireLock(locks);
-    }
-
-    @Override
-    public boolean releaseLock(BranchSession branchSession) throws TransactionException {
-        if (branchSession == null) {
-            throw new IllegalArgumentException("branchSession can't be null for memory/file locker.");
-        }
-        List<RowLock> locks = collectRowLocks(branchSession);
-        try {
-            return getLocker(branchSession).releaseLock(locks);
-        } catch (Exception t) {
-            LOGGER.error("unLock error, branchSession:{}",branchSession, t);
-            return false;
-        }
-    }
-
     @Override
     public boolean releaseGlobalSessionLock(GlobalSession globalSession) throws TransactionException {
         ArrayList<BranchSession> branchSessions = globalSession.getBranchSessions();
-        String storeMode = CONFIG.getConfig(ConfigurationKeys.STORE_MODE);
-        if (StoreMode.DB.name().equalsIgnoreCase(storeMode)) {
-            List<RowLock> locks = new ArrayList<>();
-            for (BranchSession branchSession : branchSessions) {
-                locks.addAll(collectRowLocks(branchSession));
+        boolean releaseLockResult = true;
+        for (BranchSession branchSession : branchSessions) {
+            if (!this.releaseLock(branchSession)) {
+                releaseLockResult = false;
```

Comment:
break??

Reply:
For the release of a two-phase commit, release as much as possible.